### PR TITLE
Feature/7-Basic-RabbitMQ

### PR DESCRIPTION
**Pull request by bodichka11**
To send message call post endpoint: /sendmessage/(your message)

### CODE REVIEW

**Comment by bodichka11** [comment]
https://trello.com/c/IogGP1Ud

File: backend/LeetWars.RabbitMQ/Program.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+app.MapPost("/sendmessage/{message}", (string message) =>
+{
+    using var scope = app.Services.CreateScope();
+    var _rabbitMqService = scope.ServiceProvider.GetRequiredService<IRabbitMqService>();
+    _rabbitMqService.SendMessage(message);
+
+    return Results.Ok(message);
+
+});
+app.MapGet("/", () => "Hello World!");
```
Comment (tatianahutii) [inline backend/LeetWars.RabbitMQ/Program.cs:27]: please delete test endpoints

File: backend/LeetWars.RabbitMQ/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+#Depending on the operating system of the host machines(s) that will build or run the containers, the image specified in the FROM statement may need to be changed.
+#For more information, please see https://aka.ms/containercompat
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+WORKDIR /app
+EXPOSE 80
```
Comment (tatianahutii) [inline backend/LeetWars.RabbitMQ/Dockerfile:8]: We do not need this. As Rabbit should be a separate project that will be attached to either service (not a separate service)

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Comment by bodichka11** [comment]
[Basic RabbitMQ](https://trello.com/c/IogGP1Ud/7-basic-rabbitmq)

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**